src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -4062,6 +4062,62 @@ public void testNetworksConnectContainer() throws Exception {
   }
 
   @Test
+  public void testNetworksConnectContainerWithoutForcefulDisconnectNetwork() throws Exception {
```

Comment:
I think you can leave out this test as it's almost the same as the one above and we're not testing the difference between forceful and not.